src/routes/activitypub/nodeinfo.js

```diff
@@ -1,4 +1,8 @@
-// implementation of http://nodeinfo.diaspora.software/protocol.html
+// implementation of http://nodeinfo.diaspora.software/
+// TODO: activeMonth and activeHalfyear should be dynamic, currently static
+// TODO: enable override of nodeName and nodeDescription from settings
```

Comment:
this will line up nicely with our eventual switch to storing more of the config in the database and setting everything up on the web on first-time run. I'll try to keep an eye out for that when we get there!